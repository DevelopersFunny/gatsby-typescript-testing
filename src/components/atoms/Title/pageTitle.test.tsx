import React from 'react'
import { render, screen } from '@testing-library/react'
import { PureTitle as Title } from './index'

describe('Title', () => {
  it('Renders correctly', () => {
    const pageTitle = 'Home Page'
    const data = {
      site: {
        siteMetadata: {
          title: 'Developers Funny',
        },
      },
    }
    render(<Title data={data} pageTitle={pageTitle} />)
    const titleHome = screen.getByText(/home/i)
    expect(titleHome).toMatchSnapshot()
  })
})
