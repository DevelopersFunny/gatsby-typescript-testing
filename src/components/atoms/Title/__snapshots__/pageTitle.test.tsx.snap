// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title Renders correctly 1`] = `
<title>
  Home Page
   | 
  Developers Funny
</title>
`;
